src/gt4py/next/iterator/transforms/collapse_tuple.py

```diff
@@ -56,10 +85,53 @@ class CollapseTuple(eve.PreserveLocationVisitor, eve.NodeTranslator):
       - `tuple_get(i, make_tuple(e_0, e_1, ..., e_i, ..., e_N))` -> `e_i`
     """
 
+    # TODO(tehrengruber): This Flag machanism is a little low level. What we actually want
+    #   is something like a pass manager, where for each pattern we have a corresponding
+    #   transformation, etc.
+    class Flag(enum.IntEnum):
```

Comment:
https://docs.python.org/3/library/enum.html#enum.Flag or IntFlag didn't look at the difference